# Icelandic translation for update-manager
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2015-02-18 16:10+0100\n"
"PO-Revision-Date: 2013-05-22 11:00+0000\n"
"Last-Translator: Sigurpáll Sigurðsson <Icekhaos@hotmail.com>\n"
"Language-Team: Icelandic <is@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2015-02-19 05:30+0000\n"
"X-Generator: Launchpad (build 17341)\n"
"Language: is\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:211 ../DistUpgrade/distro.py:442
#, python-format
msgid "Server for %s"
msgstr "Netþjónn fyrir %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:229 ../DistUpgrade/distro.py:235
#: ../DistUpgrade/distro.py:251
msgid "Main server"
msgstr "Aðalnetþjónn"

#: ../DistUpgrade/distro.py:255
msgid "Custom servers"
msgstr "Sérsniðnir netþjónar"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:149
msgid "Could not calculate sources.list entry"
msgstr ""

#: ../DistUpgrade/DistUpgradeAptCdrom.py:259
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Tölvan fann engar pakkaskrár, er þetta örugglega Ubuntu-diskur eða kannski "
"vitlausa gerðin?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:309
msgid "Failed to add the CD"
msgstr "Gat ekki bætt við geisladisknum"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Það kom upp villa við að bæta geisladisknum við, uppfærslan mun nú hætta. "
"Vinsamlegast tilkynntu þetta sem lús ef þetta er gildur Ubuntu-"
"geisladiskur.\n"
"\n"
"Villuskilaboðin voru:\n"
"‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:154
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Fjarlægja pakka sem er í slæmu ástandi"
msgstr[1] "Fjarlægja pakka sem eru í slæmu ástandi"

#: ../DistUpgrade/DistUpgradeCache.py:157
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Pakkinn ‚%s‘ er í lélegu ástandi og það er mikilvægt að hann verði settur "
"aftur upp, en hann fannst ekki í neinu gangasafni. Viltu eyða honum og halda "
"áfram?"
msgstr[1] ""
"Pakkarnir ‚%s‘ eru í lélegu ástandi og það er mikilvægt að þeir verði settir "
"aftur upp, en þeir fundust ekki í neinu gangasafni. Viltu eyða þeim og halda "
"áfram?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:259
msgid "The server may be overloaded"
msgstr "Netþjónninn gæti verið undir miklu álagi"

#: ../DistUpgrade/DistUpgradeCache.py:372
msgid "Broken packages"
msgstr "Bilaðir pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:373
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Kerfið þitt inniheldur bilaða pakka sem ekki var hægt að laga með þessum "
"hugbúnaði. Reyndu að laga þá með ‚synaptic‘ eða ‚apt-get‘ áður en lengra er "
"haldið."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:641
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:651
msgid "This is most likely a transient problem, please try again later."
msgstr "Þetta er væntanlega tímabundið vandamál. Reyndu aftur síðar."

#: ../DistUpgrade/DistUpgradeCache.py:654
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:659
msgid "Could not calculate the upgrade"
msgstr "Gat ekki reiknað út uppfærsluna"

#: ../DistUpgrade/DistUpgradeCache.py:712
msgid "Error authenticating some packages"
msgstr "Villa kom upp við auðkenningu nokkra pakka"

#: ../DistUpgrade/DistUpgradeCache.py:713
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Það tókst ekki að staðfesta nokkra pakka, þetta gæti verið tímabundið "
"netvandamál- reyndu aftur síðar. Að neðan er listi yfir þá pakka sem ekki "
"tókst að staðfesta."

#: ../DistUpgrade/DistUpgradeCache.py:734
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Gefin var skipun um að fjarlægja pakkann ‚%s‘  en það er bannað að fjarlægja "
"hann."

#: ../DistUpgrade/DistUpgradeCache.py:740
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Það á að fjarlægja pakkann ‚%s‘ en hann er ómissandi."

#: ../DistUpgrade/DistUpgradeCache.py:749
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Reyni að setja upp svartlistaða útgáfu ‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:868
#, python-format
msgid "Can't install '%s'"
msgstr "Gat ekki sett upp ‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:869
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:880
msgid "Can't guess meta-package"
msgstr "Get ekki giskað á lýsipakka"

#: ../DistUpgrade/DistUpgradeCache.py:881
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:118
msgid "Reading cache"
msgstr "Les skyndiminni"

#: ../DistUpgrade/DistUpgradeController.py:222
msgid "Unable to get exclusive lock"
msgstr "Fékk ekki útilokunarlæsingu"

#: ../DistUpgrade/DistUpgradeController.py:223
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Þetta þýðir líklega að annar pakkastjóri (eins og ‚apt-get‘ eða ‚aptitude‘) "
"sé í gangi. Vinsamlegast lokaðu því forriti fyrst."

#: ../DistUpgrade/DistUpgradeController.py:281
msgid "Upgrading over remote connection not supported"
msgstr "Uppfærslur yfir fjartengingu eru óstuddar"

#: ../DistUpgrade/DistUpgradeController.py:282
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:296
msgid "Continue running under SSH?"
msgstr "Viltu halda áfram að nota SSH?"

#: ../DistUpgrade/DistUpgradeController.py:297
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:311
msgid "Starting additional sshd"
msgstr "Ræsi annað sshd"

#: ../DistUpgrade/DistUpgradeController.py:312
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Til að auðvelda bata þá mun auka sshd keyra í tenginum ‚%s‘. Ef eitthvað fer "
"úrskeyðis við að keyra SSH þá er hægt að tengjast við hina tenginguna.\n"

#: ../DistUpgrade/DistUpgradeController.py:320
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:393
#: ../DistUpgrade/DistUpgradeController.py:438
msgid "Can not upgrade"
msgstr "Get ekki uppfært"

#: ../DistUpgrade/DistUpgradeController.py:394
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Þú getur ekki uppfært frá ‚%s‘ yfir í ‚%s' með þessu forriti."

#: ../DistUpgrade/DistUpgradeController.py:403
msgid "Sandbox setup failed"
msgstr "Uppsetning sandkassa mistókst"

#: ../DistUpgrade/DistUpgradeController.py:404
msgid "It was not possible to create the sandbox environment."
msgstr "Það tókst ekki að búa til sandkassann."

#: ../DistUpgrade/DistUpgradeController.py:410
msgid "Sandbox mode"
msgstr "Sandkassahamur"

#: ../DistUpgrade/DistUpgradeController.py:411
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:439
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Það er eitthvað að python uppsetningunni þinni. Reyndu að laga "
"‚/usr/bin/python‘-vísunina."

#: ../DistUpgrade/DistUpgradeController.py:468
#, python-format
msgid "Can not write to '%s'"
msgstr "Getur ekki skrifað í '%s'"

#: ../DistUpgrade/DistUpgradeController.py:469
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:480
msgid "Include latest updates from the Internet?"
msgstr "Viltu láta nýjar uppfærslur af netinu fylgja með?"

#: ../DistUpgrade/DistUpgradeController.py:481
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Uppfærslukerfið getur notað netið til að sækja nýjar uppfærslur sjálfkrafa "
"og að setja þær upp á meðan á uppfærslunni stendur. Þetta er ráðlagt ef þú "
"hefur nettengingu.\n"
"\n"
"Það tekur lengri tíma ef þú sækir uppfærslur af netinu, en á móti verður "
"tölvan af nýjustu gerð að uppfærslunni lokinni. Þú getur valið um að gera "
"þetta ekki en þú ættir að uppfæra kerfið þegar samt sem áður.\n"
"Netið verður ekki notað ef þú velur að gera þetta ekki."

#: ../DistUpgrade/DistUpgradeController.py:641
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:726
#, python-format
msgid "disabled on upgrade to %s"
msgstr "óvirkt að uppfæra til %s"

#: ../DistUpgrade/DistUpgradeController.py:761
msgid "No valid mirror found"
msgstr "Enginn spegill fannst"

#: ../DistUpgrade/DistUpgradeController.py:762
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:782
msgid "Generate default sources?"
msgstr "Búa til sjálfgefnar uppsprettur?"

#: ../DistUpgrade/DistUpgradeController.py:783
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Engin færsla fyrir ‚%s‘ fannst í ‚sources.list‘ skránni þinni.\n"
"\n"
"Viltu að tölvan bæti inn færslu fyrir ‚%s‘ eða viltu velja ‚Nei‘ og hætta "
"við uppfærsluna?"

#: ../DistUpgrade/DistUpgradeController.py:818
#: ../DistUpgrade/DistUpgradeController.py:824
msgid "Repository information invalid"
msgstr "Ógildar upplýsingar um gagnahirslu"

#: ../DistUpgrade/DistUpgradeController.py:819
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:825
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:834
msgid "Third party sources disabled"
msgstr "Frumgögn frá þriðja aðila voru gerð óvirk"

#: ../DistUpgrade/DistUpgradeController.py:835
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Einhverjir innslættir þriðju aðila voru gerðir óvirkir í skránni "
"‚sources.list‘. Þú getur virkjað þá seinna með tólinu ‚software-properties‘ "
"eða með pakkastjóranum þínum."

#: ../DistUpgrade/DistUpgradeController.py:877
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakkinn er í versnandi ásigkomulagi"
msgstr[1] "Pakkarnir eru í versnandi ásigkomulagi"

#: ../DistUpgrade/DistUpgradeController.py:880
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Það þarf að setja pakkann ‚%s‘ upp aftur þar sem hann er í ójafnvægi en hann "
"fannst hins vegar ekki í gagnahirslunni. Þú verður því að setja hann upp "
"handvirkt eða fjarlægja af tölvunni."
msgstr[1] ""
"Það þarf að setja pakkana ‚%s‘ upp aftur þar sem þeir eru í ójafnvægi en "
"þeir fundust hins vegar ekki í gagnahirslunni. Þú verður því að setja þá upp "
"handvirkt eða fjarlægja af tölvunni."

#: ../DistUpgrade/DistUpgradeController.py:931
msgid "Error during update"
msgstr "Villa kom upp í uppfærslu"

#: ../DistUpgrade/DistUpgradeController.py:932
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Það kom upp villa við að uppfæra tölvuna. Þetta er oftast vandamál með netið "
"þannig að best væri að athuga nettenginguna og reyna aftur."

#: ../DistUpgrade/DistUpgradeController.py:941
msgid "Not enough free disk space"
msgstr "Ekki er nóg pláss á tölvunni"

#: ../DistUpgrade/DistUpgradeController.py:942
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:971
#: ../DistUpgrade/DistUpgradeController.py:1779
msgid "Calculating the changes"
msgstr "Reikna út breytingar"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1003
msgid "Do you want to start the upgrade?"
msgstr "Viltu hefja uppfærsluna?"

#: ../DistUpgrade/DistUpgradeController.py:1076
msgid "Upgrade canceled"
msgstr "Það var hætt við uppfærsluna"

#: ../DistUpgrade/DistUpgradeController.py:1077
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1083
#: ../DistUpgrade/DistUpgradeController.py:1208
msgid "Could not download the upgrades"
msgstr "Gat ekki sótt uppfærslurnar"

#: ../DistUpgrade/DistUpgradeController.py:1084
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1159
#: ../DistUpgrade/DistUpgradeController.py:1196
#: ../DistUpgrade/DistUpgradeController.py:1301
msgid "Error during commit"
msgstr "Villa við staðfestingu"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1161
#: ../DistUpgrade/DistUpgradeController.py:1198
#: ../DistUpgrade/DistUpgradeController.py:1340
msgid "Restoring original system state"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1162
#: ../DistUpgrade/DistUpgradeController.py:1177
#: ../DistUpgrade/DistUpgradeController.py:1199
msgid "Could not install the upgrades"
msgstr "Gat ekki sett upp uppfærslurnar"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1167
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1172
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug "
"and attach the files in /var/log/dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1209
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1289
msgid "Remove obsolete packages?"
msgstr "Viltu fjarlægja úrelta pakka?"

#: ../DistUpgrade/DistUpgradeController.py:1290
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Halda"

#: ../DistUpgrade/DistUpgradeController.py:1290
msgid "_Remove"
msgstr "_Fjarlægja"

#: ../DistUpgrade/DistUpgradeController.py:1302
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Það kom upp vandamál við að hreinsa til. Lestu skilaboðin fyrir neðan til að "
"fá frekari upplýsingar. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1378
msgid "Required depends is not installed"
msgstr "Það er ekki búið að setja upp nauðsynleg ákvæði"

#: ../DistUpgrade/DistUpgradeController.py:1379
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Það er ekki búið að setja upp ákvæðið ‚%s‘ sem er nauðsynlegt. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1646
#: ../DistUpgrade/DistUpgradeController.py:1733
msgid "Checking package manager"
msgstr "Athuga pakkastjórann"

#: ../DistUpgrade/DistUpgradeController.py:1652
#: ../DistUpgrade/DistUpgradeController.py:1658
msgid "Preparing the upgrade failed"
msgstr "Undirbúningur fyrir uppfærslu mistókst"

#: ../DistUpgrade/DistUpgradeController.py:1653
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1659
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1676
#: ../DistUpgrade/DistUpgradeController.py:1686
msgid "Getting upgrade prerequisites failed"
msgstr "Það tókst ekki að útvega forsendur uppfærslunnar"

#: ../DistUpgrade/DistUpgradeController.py:1677
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1687
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1717
msgid "Updating repository information"
msgstr "Uppfæri upplýsingar um gagnahirslur"

#: ../DistUpgrade/DistUpgradeController.py:1724
msgid "Failed to add the cdrom"
msgstr "Gat ekki bætt við geisladisknum"

#: ../DistUpgrade/DistUpgradeController.py:1725
msgid "Sorry, adding the cdrom was not successful."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1756
msgid "Invalid package information"
msgstr "Ógildar pakkaupplýsingar"

#: ../DistUpgrade/DistUpgradeController.py:1757
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1785
#: ../DistUpgrade/DistUpgradeController.py:1841
msgid "Fetching"
msgstr "Sæki"

#: ../DistUpgrade/DistUpgradeController.py:1791
#: ../DistUpgrade/DistUpgradeController.py:1845
msgid "Upgrading"
msgstr "Uppfæri"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1796
#: ../DistUpgrade/DistUpgradeController.py:1847
#: ../DistUpgrade/DistUpgradeController.py:1854
#: ../DistUpgrade/DistUpgradeController.py:1865
msgid "Upgrade complete"
msgstr "Uppfærslu lokið"

#: ../DistUpgrade/DistUpgradeController.py:1797
#: ../DistUpgrade/DistUpgradeController.py:1848
#: ../DistUpgrade/DistUpgradeController.py:1855
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1804
msgid "Searching for obsolete software"
msgstr "Leita að úreltum hugbúnaði"

#: ../DistUpgrade/DistUpgradeController.py:1817
msgid "System upgrade is complete."
msgstr "Stýrikerfisuppfærslu er lokið."

#: ../DistUpgrade/DistUpgradeController.py:1866
msgid "The partial upgrade was completed."
msgstr "Takmarkaðri uppfærslu er lokið."

#: ../DistUpgrade/DistUpgradeFetcher.py:118
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
msgid "Could not find the release notes"
msgstr "Fann ekki upplýsingar um útgáfu"

#: ../DistUpgrade/DistUpgradeFetcher.py:119
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:157
msgid "The server may be overloaded. "
msgstr "Það er mögulega mikið álag á netþjóninum. "

#: ../DistUpgrade/DistUpgradeFetcher.py:131
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:161
msgid "Could not download the release notes"
msgstr "Gat ekki sótt upplýsingar um útgáfuna"

#: ../DistUpgrade/DistUpgradeFetcher.py:132
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:162
msgid "Please check your internet connection."
msgstr "Vinsamlegast athugaðu nettenginguna þína"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:137
#, python-format
msgid "extracting '%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:158
#: ../DistUpgrade/DistUpgradeFetcherCore.py:159
msgid "Could not run the upgrade tool"
msgstr "Gat ekki keyrt uppfærslutólið"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:160
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:237
msgid "Upgrade tool signature"
msgstr "Auðkenni uppfærslutóls"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid "Upgrade tool"
msgstr "Uppfærsluforrit"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid "Failed to fetch"
msgstr "Ekki tókst að sækja"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Það tókst ekki að sækja uppfærsluna. Þetta gæti verið vandamál með netið. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:283
msgid "Authentication failed"
msgstr "Auðkenning mistókst"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:284
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Það tókst ekki að auðkenna uppfærsluna. Þetta er mögulega vandamál við "
"nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:289
msgid "Failed to extract"
msgstr "Gat ekki afþjappað"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:290
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Það mistókst að nálgast upplýsingarnar um uppfærsluna. Þetta er mögulega "
"vandamál við nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:295
msgid "Verification failed"
msgstr "Staðfesting mistókst"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:296
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Það tókst ekki að staðfesta uppfærsluna. Það gæti verið vandamál með "
"nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:310
#: ../DistUpgrade/DistUpgradeFetcherCore.py:316
msgid "Can not run the upgrade"
msgstr "Get ekki keyrt uppfærsluna"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:311
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:317
#, python-format
msgid "The error message is '%s'."
msgstr "Villuskilaboðið er ‚%s‘."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:137
msgid "&Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:140
#: ../DistUpgrade/DistUpgradeViewKDE.py:579
msgid "&Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:142
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Upplýsingar um útgáfu"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:198
msgid "Upgrade"
msgstr "Uppfæra"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:213
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:226
msgid "Downloading additional package files..."
msgstr "Sæki aukapakka..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Skrá %s af %s á %sB/sek."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:232
#, python-format
msgid "File %s of %s"
msgstr "Skrá %s af %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:239
#: ../DistUpgrade/DistUpgradeViewGtk3.py:123
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Settu ‚%s‘ í drifið ‚%s‘"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:241
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:246
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:218
msgid "Media Change"
msgstr "Breyta miðli"

#: ../DistUpgrade/DistUpgradeQuirks.py:164
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:166
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D Do you still "
"want to continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:190
msgid ""
"Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:192
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:212
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Þá má vera að brellur, leikir og önnur forrit sem nota skjákortið mikið "
"virki ekki eins vel að uppfærslu lokinni."

#: ../DistUpgrade/DistUpgradeQuirks.py:216
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Þessi tölva er sem stendur að notast við AMD ‚fglrx‘-rekilinn fyrir grafík. "
"Það er ekki til nein útgáfa af þessum rekli sem virkar með vélbúnaði þínum í "
"Ubuntu 10.04 LTS.\n"
"\n"
"Viltu halda áfram?"

#: ../DistUpgrade/DistUpgradeQuirks.py:251
msgid "No i686 CPU"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:252
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:289
msgid "No ARMv6 CPU"
msgstr "Enginn ARMv6-örgjörvi"

#: ../DistUpgrade/DistUpgradeQuirks.py:290
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Tölva þín notar ARM-örgjörva sem er eldri en ARMv6 (útgáfa 6). Allir pakkar "
"í karmic þurfa að lágmarki örgjörva af útgáfu ARMv6. Það er ekki hægt að "
"uppfæra stýrikerfi þitt yfir í nýja útgáfu af Ubuntu með þessum vélbúnaði."

#: ../DistUpgrade/DistUpgradeQuirks.py:310
msgid "No init available"
msgstr "‚init‘ fannst ekki"

#: ../DistUpgrade/DistUpgradeQuirks.py:311
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:412
msgid "PAE not enabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:413
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Sandbox upgrade using aufs"
msgstr "Uppfærsla í sandkassanum með ‚aufs‘"

#: ../DistUpgrade/DistUpgradeMain.py:61
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Nota þessa slóð til að leita að geisladiski með pökkum sem hægt er að "
"uppfæra með"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Nota framenda. Nú þegar tiltækir: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "*DEPRECATED* this option will be ignored"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Ekki uppfæra allt (mun ekki skrifa yfir ‚sources.list‘)"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "Disable GNU screen support"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Set datadir"
msgstr "Ákveða ‚datadir‘"

#: ../DistUpgrade/DistUpgradeMain.py:81
msgid "Upgrade to the development release"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:144
#: ../DistUpgrade/DistUpgradeViewKDE.py:231
msgid "Fetching is complete"
msgstr "Búið að sækja"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:159
#: ../DistUpgrade/DistUpgradeViewKDE.py:247
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Sæki skrá %li af %li á %sB/sek."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewGtk3.py:319
#: ../DistUpgrade/DistUpgradeViewKDE.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:398
#, python-format
msgid "About %s remaining"
msgstr "Um það bil %s eftir"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:164
#: ../DistUpgrade/DistUpgradeViewKDE.py:250
#, python-format
msgid "Fetching file %li of %li"
msgstr "Sæki skrá %li af %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:196
#: ../DistUpgrade/DistUpgradeViewKDE.py:288
msgid "Applying changes"
msgstr "Virki breytingar"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:222
#: ../DistUpgrade/DistUpgradeViewKDE.py:301
msgid "dependency problems - leaving unconfigured"
msgstr "vandamál með ákvæði - skil eftir óstillt"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:227
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
#, python-format
msgid "Could not install '%s'"
msgstr "Gat ekki sett upp ‚%s‘"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:228
#: ../DistUpgrade/DistUpgradeViewKDE.py:304
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Uppfærslan mun halda áfram en pakkinn ‚%s‘ er líklega í ólagi. Vinsamlegast "
"sentu inn upplýsingar um þessa bilun."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:245
#: ../DistUpgrade/DistUpgradeViewKDE.py:326
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Skipta út sérsniðnu stillingarskráni\n"
" ‚%s‘?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:246
#: ../DistUpgrade/DistUpgradeViewKDE.py:327
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Þú munt tapa öllu sem þú hefur breytt í þessari stillingarskrá ef þú ákveður "
"að skipta henni út fyrir nýja útgáfu."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:266
#: ../DistUpgrade/DistUpgradeViewKDE.py:350
msgid "The 'diff' command was not found"
msgstr "Skipunin ‚diff‘ fannst ekki"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
#: ../DistUpgrade/DistUpgradeViewText.py:135
msgid "A fatal error occurred"
msgstr "Það kom upp grafalvarleg villa"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:497
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:524
msgid "Ctrl-c pressed"
msgstr "Ýtt á ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:525
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Þetta mun stöðva aðgerðina og skilja kerfið eftir bilað. Ertu viss um að þú "
"viljir halda áfram?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:663
msgid "To prevent data loss close all open applications and documents."
msgstr "Til að fyrirbyggja tap á gögnum skaltu loka öllum forritum og skrám."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:677
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:678
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:679
#, python-format
msgid "Remove (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:680
#, python-format
msgid "No longer needed (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:681
#, python-format
msgid "Install (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:682
#, python-format
msgid "Upgrade (%s)"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:362
msgid "Show Difference >>>"
msgstr "Sýna mun >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:365
msgid "<<< Hide Difference"
msgstr "<<< Fela mun"

#: ../DistUpgrade/DistUpgradeViewKDE.py:565
msgid "Error"
msgstr "Villa"

#: ../DistUpgrade/DistUpgradeViewKDE.py:583
#: ../DistUpgrade/DistUpgradeViewKDE.py:817
msgid "&Close"
msgstr "&Loka"

#: ../DistUpgrade/DistUpgradeViewKDE.py:617
msgid "Show Terminal >>>"
msgstr "Sýna útstöðina >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:620
msgid "<<< Hide Terminal"
msgstr "<<< Fela útstöðina"

#: ../DistUpgrade/DistUpgradeViewKDE.py:699
msgid "Information"
msgstr "Upplýsingar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:751
msgid "&Start Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
#: ../DistUpgrade/DistUpgradeViewKDE.py:798
#: ../DistUpgrade/DistUpgradeViewKDE.py:803
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Frekari upplýsingar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:779
#, python-format
msgid "No longer supported %s"
msgstr "Ekki lengur stutt %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#, python-format
msgid "Remove %s"
msgstr "Fjarlægja %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#: ../DistUpgrade/DistUpgradeViewText.py:233
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Fjarlægja (var sett upp sjálfvirkt) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#, python-format
msgid "Install %s"
msgstr "Setja upp %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:787
#, python-format
msgid "Upgrade %s"
msgstr "Uppfæra %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:813
#: ../DistUpgrade/DistUpgradeViewText.py:281
msgid "Restart required"
msgstr "Það þarf að endurræsa tölvuna"

#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Endurræsa kerfið til að ljúka uppfærslunni</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:816
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Endurræsa núna"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Viltu hætta við uppfærsluna?</big></b>\n"
"\n"
"Það getur verið að tölvan þín verði ónothæf ef þú hættir núna. Það er "
"stranglega mælt með því að halda áfram."

#: ../DistUpgrade/DistUpgradeViewKDE.py:838
msgid "Cancel Upgrade?"
msgstr "Viltu hætta við uppfærsluna?"

#: ../DistUpgrade/DistUpgradeView.py:81
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dagur"
msgstr[1] "%li dagar"

#: ../DistUpgrade/DistUpgradeView.py:83
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li klukkutími"
msgstr[1] "%li klukkutímar"

#: ../DistUpgrade/DistUpgradeView.py:85
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li mínúta"
msgstr[1] "%li mínútur"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekúnda"
msgstr[1] "%li sekúndur"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:102
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:120
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:171
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Þetta niðurhal mun taka um %s með 1Mbit DSL tengingu og um %s með 56k "
"símalínutengingu."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:175
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Þetta mun taka um %s með þinni tengingu. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:280
#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Undirbý að uppfæra"

#: ../DistUpgrade/DistUpgradeView.py:281
msgid "Getting new software channels"
msgstr "Sæki nýjar hugbúnaðarrásir"

#: ../DistUpgrade/DistUpgradeView.py:282
#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Sæki nýja pakka"

#: ../DistUpgrade/DistUpgradeView.py:283
#: ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Set upp uppfærslurnar"

#: ../DistUpgrade/DistUpgradeView.py:284
#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Hreinsa til"

#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
msgstr[1] ""

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:381
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakki mun vera fjarlægður."
msgstr[1] "%d pakkar munu vera fjarlægðir."

#: ../DistUpgrade/DistUpgradeView.py:386
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nýr pakki mun vera uppsettur."
msgstr[1] "%d nýir pakkar munu vera uppsettir."

#: ../DistUpgrade/DistUpgradeView.py:392
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakki mun vera uppfærður."
msgstr[1] "%d pakkar munu vera uppfærðir."

#: ../DistUpgrade/DistUpgradeView.py:400
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Þú þarft að sækja samtals %s. "

#: ../DistUpgrade/DistUpgradeView.py:405
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:409
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:414
msgid "Removing the packages can take several hours. "
msgstr ""

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:419
msgid "The software on this computer is up to date."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:420
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Það eru engar uppfærslur fáanlegar fyrir kerfið þitt. Uppfærslu verður nú "
"hætt."

#: ../DistUpgrade/DistUpgradeView.py:433
msgid "Reboot required"
msgstr "Þú þarft að endurræsa tölvuna"

#: ../DistUpgrade/DistUpgradeView.py:434
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Uppfærslum er lokið og nú þarf að endurræsa tölvuna. Viltu gera það núna?"

#: ../DistUpgrade/DistUpgradeViewText.py:78
#, python-format
msgid "Progress: [%3i%%]"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:136
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:160
msgid "Aborting"
msgstr "Hætti við"

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Demoted:\n"
msgstr "Lækkað í tign:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:172
msgid "To continue please press [ENTER]"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:209
#: ../DistUpgrade/DistUpgradeViewText.py:247
#: ../DistUpgrade/DistUpgradeViewText.py:254
msgid "Continue [yN] "
msgstr "Viltu halda áfram [jN] "

#: ../DistUpgrade/DistUpgradeViewText.py:209
#: ../DistUpgrade/DistUpgradeViewText.py:247
msgid "Details [d]"
msgstr "Frekari upplýsingar [u]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:213
#: ../DistUpgrade/DistUpgradeViewText.py:257
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:216
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:219
msgid "d"
msgstr "u"

#: ../DistUpgrade/DistUpgradeViewText.py:224
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ekki lengur stutt: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:229
#, python-format
msgid "Remove: %s\n"
msgstr "Fjarlægja: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Install: %s\n"
msgstr "Setja upp: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:244
#, python-format
msgid "Upgrade: %s\n"
msgstr "Uppfæra: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [Yn] "
msgstr "Viltu halda áfram [Jn] "

#: ../DistUpgrade/DistUpgradeViewText.py:282
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Til að ljúka við uppfærsluna þarf að endurræsa tölvuna.\n"
"Ef þú velur ‚j‘ mun hún endurræsa sig."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Sæki skrá %(current)li af %(total)li á %(speed)s/sek."

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Sæki skrá %(current)li af %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Skoða framvindu einstakra skráa"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Hætta við uppfærslu"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Halda uppfærslu áfram"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Hætta við þá uppfærslu sem er í gangi?</big></b>\n"
" \n"
"Kerfið gæti orðið ónothæft ef þú hættir við uppfærlsuna núna. Það er "
"stranglega mælt með að þú haldir áfram með uppfærsluna."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Hefja uppfærslu"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Skipta út"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Munur á skránum"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Tilkynna villu"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Halda áfram"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Hefja uppfærslu?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Endurræstu tölvuna til að ljúka uppfærslunni</big></b>\n"
" \n"
"Mundu eftir því að vista öll gögn áður en þú heldur áfram."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Kerfisuppfærsla"

#: ../data/gtkbuilder/DistUpgrade.ui.h:19
msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Ákveð nýjar hugbúnaðarrásir"

#: ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Endurræsi tölvuna"

#: ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Útstöð"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Uppfæra"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Ný útgáfa Ubuntu er tiltæk. Viltu uppfæra í hana núna?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Ekki uppfæra"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Spyrja síðar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Já, uppfæra núna"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Þú hefur hafnað uppfærslu í nýja útgáfu af Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr ""

#: ../do-partial-upgrade:74 ../do-release-upgrade:81
msgid "Show version and exit"
msgstr "Sýna útgáfu og hætta"

#: ../do-partial-upgrade:77 ../do-release-upgrade:88
msgid "Directory that contains the data files"
msgstr "Mappan sem inniheldur gagnaskrárnar"

#: ../do-partial-upgrade:80 ../do-release-upgrade:101
msgid "Run the specified frontend"
msgstr "Keyra tiltekið viðmót"

#: ../do-partial-upgrade:103
msgid "Running partial upgrade"
msgstr "Keyra hlutauppfærslu"

#: ../do-release-upgrade:31 ../do-release-upgrade:43
msgid "Downloading the release upgrade tool"
msgstr "Sækir uppfærslutólið fyrir útgáfuna"

#: ../do-release-upgrade:84 ../check-new-release-gtk:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Athuga hvort hægt sé að uppfæra í nýjustu þróunarútgáfuna"

#: ../do-release-upgrade:91 ../check-new-release-gtk:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""

#: ../do-release-upgrade:95
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""

#: ../do-release-upgrade:103
msgid "Test upgrade with a sandbox aufs overlay"
msgstr ""

#: ../do-release-upgrade:106
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""

#: ../do-release-upgrade:120
msgid "The options --devel-release and --proposed are"
msgstr ""

#: ../do-release-upgrade:121
msgid "mutually exclusive. Please use only one of them."
msgstr ""

#: ../do-release-upgrade:125
msgid "Checking for a new Ubuntu release"
msgstr ""

#: ../do-release-upgrade:137
msgid "Your Ubuntu release is not supported anymore."
msgstr ""

#: ../do-release-upgrade:138
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""

#: ../do-release-upgrade:144
msgid "No new release found"
msgstr "Engar nýjar útgáfur fundust"

#: ../do-release-upgrade:149
msgid "Release upgrade not possible right now"
msgstr "Útgáfuuppfærsla ekki möguleg í augnablikinu."

#: ../do-release-upgrade:150
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""

#: ../do-release-upgrade:156
#, c-format
msgid "New release '%s' available."
msgstr "Nýja útgáfan ‚%s‘ fannst."

#: ../do-release-upgrade:157
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Keyrðu ‚do-release-upgrade‘ til að hefja uppfærsluna."

#: ../check-new-release-gtk:85
msgid "Software Updater"
msgstr ""

#: ../check-new-release-gtk:113
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Uppfærsla í Ubuntu %(version)s er tiltæk"

#: ../check-new-release-gtk:144
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Þú hefur hafnað uppfærslu í Ubuntu %s"

#: ../check-new-release-gtk:186
msgid "Add debug output"
msgstr ""

#, python-format
#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Villa kom upp við að reikna út uppfærsluna:\n"
#~ "%s\n"
#~ "\n"
#~ " Nokkrar ástæður geta ollið þessu:\n"
#~ " * Þú ert að uppfæra yfir í prufuútgáfu af Ubuntu\n"
#~ " * Þú ert að nota prufuútgáfu af Ubuntu\n"
#~ " * Þú hefur náð í hugbúnaðarpakka sem eru ekki á vegum Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect which "
#~ "version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "Kerfið þitt inniheldur hvorki pakkann ‚ubuntu-desktop‘, ‚kubuntu-desktop‘, "
#~ "‚xubuntu-desktop‘ né ‚edubuntu-desktop‘- og ekki tókst að bera kennsl á "
#~ "hvaða tegund af Ubuntu þú ert að nota.\n"
#~ " Settu upp einn af áðurnefndum pökkum með því að nota Synaptic-pakkastjórann "
#~ "eða með því að nota apt-get áður en þú heldur áfram."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "Pakkinn 'debsig-verify' er uppsettur"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
#~ "run the upgrade again."
#~ msgstr ""
#~ "Uppfærslan getur ekki haldið áfram á meðan þessi pakki er á tölvunni.\n"
#~ "Fjarlægðu hann með synaptic pakkastjóranum eða með því að nota ‚apt-get "
#~ "remove debsig-verify‘ fyrst og keyra uppfærsluna aftur."

#~ msgid "evms in use"
#~ msgstr "‚evms‘ í notkun"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "Kerfið þitt notar ‚evms‘-gagnamiðilsstjóra í ‚/proc/mounts‘-möppunni. "
#~ "Hugbúnaðurinn ‚evms‘ er ekki lengur studdur, slökktu á honum og keyrðu "
#~ "uppfærsluna aftur að því loknu."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Þessi tölva er sem stendur að notast við NVIDIA ‚nvidia‘-rekilinn fyrir "
#~ "grafík. Það er ekki til nein útgáfa af reklinum sem virkar með skjákortinu "
#~ "þínu í Ubuntu 10.04 LTS\n"
#~ "\n"
#~ "Viltu halda áfram?"
